Dockerfile

```diff
@@ -12,8 +11,7 @@ RUN apk update -f \
   tzdata \
   oath-toolkit-oathtool \
   tar \
-  libidn \
-  && rm -rf /var/cache/apk/*
```

Comment:
why is this line removed ?

Reply:
newer docker images for most OSs do this via internal scripts.